README.md

```diff
@@ -476,8 +479,8 @@ This table describes the list of available parameters for Helm Chart.
 | `persistence.annotations`            | Annotations for Persistence Volume Claim (PVC)                                                                 | `{}`                                 |
 | `persistence.accessModes`            | Access mode for persistence volume                                                                             | `["ReadWriteOnce"]`                  |
 | `persistence.mountPath`              | The path for the mounted persistence data volume                                                               | `/var/opt/MarkLogic`                 |
-| `extraVolumes`                       | Extra list of additional volumes for MarkLogic statefulset                                                     | `[]`                                 |
-| `extraVolumeMounts`                  | Extra list of additional volumeMounts for MarkLogic container                                                  | `[]`                                 |
+| `additionalVolumes`                  | Additional list of additional volumes for MarkLogic statefulset                                                | `[]`                                 |
+| `AdditionalVolumeMounts`             | Additional list of additional volumeMounts for MarkLogic container                                             | `[]`                                 |
```

Comment:
Fix the case "additionalVolumeMounts"

List of mount points for the additional volumes to add to the MarkLogic containers.